frontend/inverted-index.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * InvertedIndex class with constructor
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.allFiles = {};
+    this.index = {};
+    this.docNumber = {};
```

Comment:
**Nitpick**
do not use abbreviations for variable naming
https://github.com/andela/code-review-guidelines/tree/master/coding-style-guides